src/v1/destinations/iterable/types.ts

```diff
@@ -0,0 +1,57 @@
+type FailedUpdates = {
+  invalidEmails?: string[];
+  invalidUserIds?: string[];
+  notFoundEmails?: string[];
+  notFoundUserIds?: string[];
+  invalidDataEmails?: string[];
+  invalidDataUserIds?: string[];
+  conflictEmails?: string[];
+  conflictUserIds?: string[];
+  forgottenEmails?: string[];
+  forgottenUserIds?: string[];
+};
+
+export type GeneralApiResponse = {
+  msg?: string;
+  code?: string;
+  params?: Record<string, unknown>;
+  successCount?: number;
+  failCount?: number;
+  invalidEmails?: string[];
+  invalidUserIds?: string[];
+  filteredOutFields?: string[];
+  createdFields?: string[];
+  disallowedEventNames?: string[];
+  failedUpdates?: FailedUpdates;
+};
+
+export type DestinationResponse = {
+  status: number;
+  response: GeneralApiResponse;
+};
+
+export type ResponseParams = {
```

Comment:
Why do we need a specific type for Iterable ? We should not be adding type for a single integration changes